modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -63,7 +67,7 @@ endif::[]
 
 1. Egress router for OVN-Kubernetes supports only redirect mode.
 
-2. IPv6 is supported only on bare metal, vSphere, {ibm-power-name}, {ibm-z-name}, and Red Hat OpenStack clusters.
+2. IPv6 single-stack networking is only supported on bare metal, {ibm-power-name}, {ibm-z-name}, and {rh-openstack-first} clusters. You must enable IPv6 access for your image registry to prevent network connectivity issues when you need to pull images from the registry.
```

Comment:
This is not equivalent. I think the intent of the "IPv6" support was actually dual stack. Single stack only works on baremetal right now.

I think this needs to be rewritten to say this section is for dual stack and then the point below should be changed to say that IPv6 single stack is supported only on baremetal, instead of the exclusion list (which is technically not even complete now since IPv6 isn't supported on the cloud platforms either at this point).